src/components/Editable.js

```diff
@@ -236,16 +238,21 @@ export const Editable = connect()(({ focus, itemsRanked, contextChain, showConte
       // the data will be available as text/plain or text/html
       // this reflects the format of the source data more than the actual contents
       // text/plain may contain text that ultimately looks like html (contains <li>) and should be parsed as html
-      const plainText = e.clipboardData.getData('text/plain')
-      const htmlText = e.clipboardData.getData('text/html')
+      let plainText = e.clipboardData.getData('text/plain')
+      let htmlText = e.clipboardData.getData('text/html')
```

Comment:
Please read the [Contributing Guidelines](https://github.com/cybersemics/em/blob/dev/CONTRIBUTING.md). `let` is not allowed on this project. You must create a new variable and use `const`.